CHANGELOG.md

```diff
@@ -1,3 +1,47 @@
+
+#### Android
+* New capabilities:
+  * `remoteAppsCacheLimit`: sets the limit for how many APKs will be cached on a device [#523](https://github.com/appium/appium-android-driver/pull/523)
+  * `chromedriverPorts`: allows specifying multiple ports or a range of Chromedriver ports to use for web tests [#529](https://github.com/appium/appium-android-driver/pull/529)
+  * `buildToolsVersion`: allows you to set the Android `build-tools` version to be something different than the default, which is to use the most recent version [#532](https://github.com/appium/appium-android-driver/pull/532)
+* Fixes:
+  * Emulators have a bug where they sometimes go offline when root/unroot is called. Only affected unrooted emulators. Workaround is to check if a device went offline after root/unroot was called and then restarting the ADB server [#443](https://github.com/appium/appium-adb/pull/443)
+  * Calls to `mobile:` endpoints weren't being called in web context. Defaults now is that in a web context, the native mobile endpoint is always called [#527](https://github.com/appium/appium-android-driver/pull/527)
+  * `pushFile` was not working on some later Android SDK's due to permission errors [#439](https://github.com/appium/appium-adb/pull/439)
+  * `network_connection` was not usable from web context [#531](https://github.com/appium/appium-android-driver/pull/531)
+  * Default values in caps not being set correctly [#436](https://github.com/appium/appium-adb/pull/436)
+* No longer uninstalls apps when session is terminated if `dontStopAppOnReset` is set [#530](https://github.com/appium/appium-android-driver/pull/530)
+* Allow touch actions in a web context. Only works for absolute coordinates. JSONWP only, does not affect behaviour of W3C Actions implementation [#534](https://github.com/appium/appium-android-driver/pull/534/files)
+* Update Chromedriver to 73.0.0 [#318](https://github.com/appium/appium-base-driver/pull/318)
+
+#### Android (UiAutomator2)
+* Use UiAutomator's screenshot method when default screenshoter fails [#264](https://github.com/appium/appium-uiautomator2-server/pull/264)
+* Add detailed network information to the device information endpoint (appium/device/info) [#265](https://github.com/appium/appium-uiautomator2-server/pull/265). Addresses [Issue #12502](https://github.com/appium/appium/issues/12502)
+
+#### Android (Espresso)
+* Fixes: 
+  * Changed incorrect naming of Espresso argument `installTimeout` to the correct `androidInstallTimeout` [#426](https://github.com/appium/appium-espresso-driver/pull/426/files)
+  * Use ADB instead of Espresso to verify activities for better reliability [#425](https://github.com/appium/appium-espresso-driver/pull/425/files)
+
+#### iOS
+* Support webview testing for real devices running iOS 12.2+ (1.12.1 already added support for iOS Simulators) [#122](https://github.com/appium/appium-remote-debugger/pull/122)
```

Comment:
Is it better to address the latest https://github.com/google/ios-webkit-debug-proxy/releases/tag/v1.8.4 , too?

Reply:
Sure. What should the message say?

Reply:
Something like... _Make sure your ios-webkit-debug-proxy is up-to-date for real devices._